<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WKWIN7028371" name="TestRunner" tests="2" failures="1" timestamp="2021-03-27T00:57:52 IST" time="70.931" errors="0">
  <testcase name="runScenario" time="47.186" classname="TestRunner"/>
  <testcase name="runScenario" time="23.745" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="-- Invalid credentials should be displayed -- did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: -- Invalid credentials should be displayed -- did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.payoneer.cloud.box.steps.SignInPageStepDefs.verify_invalid_cred_message(SignInPageStepDefs.java:61)
at âœ½.I verify user gets invalid credential message(src/test/resources/features/01_Login.feature:18)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner -->
